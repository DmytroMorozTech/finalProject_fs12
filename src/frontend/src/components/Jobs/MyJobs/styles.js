import { makeStyles } from '@material-ui/core/styles'

export default makeStyles(() => ({

  myJobsPage: {
    marginTop: '75px'
  }
}))
