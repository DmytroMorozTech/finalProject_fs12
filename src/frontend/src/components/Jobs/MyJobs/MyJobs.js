import styles from './styles'
import { Container } from '@material-ui/core'
import Grid from '@material-ui/core/Grid'
import React from 'react'
import MyJobsLeft from './MyJobsLeft/MyJobsLeft'
import MyJobsMain from './MyJobsMain/MyJobsMain'
import MyJobsRight from './MyJobsRight/MyJobsRight'

function MyJobs () {
  const classes = styles()

  return (
    <Container className={classes.myJobsPage} maxWidth={'lg'}>
      <Grid container spacing={2} alignItems="flex-start" justifyContent="center">

        <Grid item xs={12} sm={10} md={2} lg={2} xl={2}>
          <MyJobsLeft/>
        </Grid>

        <Grid item xs={12} sm={10} md={5} lg={6} xl={6}>
          <MyJobsMain/>
        </Grid>

        <Grid item xs={12} sm={10} md={3} lg={3} xl={3}>
          <MyJobsRight/>
        </Grid>

      </Grid>
    </Container>
  )
}

export default MyJobs
