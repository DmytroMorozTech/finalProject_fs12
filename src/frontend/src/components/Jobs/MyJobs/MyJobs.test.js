import { shallow } from 'enzyme'
import React from 'react'
import MyJobs from './MyJobs'

jest.mock('./styles', () => () => ({}))
jest.mock('react-router-dom', () => ({
  Link: 'a'
}))

const setUp = () => shallow(<MyJobs/>)

describe('should render MyJobs component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
