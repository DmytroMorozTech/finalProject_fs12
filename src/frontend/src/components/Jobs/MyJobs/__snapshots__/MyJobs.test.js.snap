// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MyJobs component to match snapshot 1`] = `
<WithStyles(ForwardRef(Container))
  maxWidth="lg"
>
  <WithStyles(ForwardRef(Grid))
    alignItems="flex-start"
    container={true}
    justifyContent="center"
    spacing={2}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={2}
      md={2}
      sm={10}
      xl={2}
      xs={12}
    >
      <MyJobsLeft />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={6}
      md={5}
      sm={10}
      xl={6}
      xs={12}
    >
      <MyJobsMain />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={3}
      md={3}
      sm={10}
      xl={3}
      xs={12}
    >
      <MyJobsRight />
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Container))>
`;
